agenta-cli/agenta/cli/main.py

```diff
@@ -126,6 +126,7 @@ def init(app_name: str):
             "app_name": app_name,
             "app_id": app_id,
             "backend_host": backend_host,
+            "tracking_enabled": True,
```

Comment:
Instead of using the local config.toml, I think we should use the global config.toml we use to save the api key (`~/.agenta/config.toml`). This would allow the user to disable tracking for all applications at one and not do it for each again.

Reply:
Thank you!